pkg/piecestore/psserver/psdb/psdb.go

```diff
@@ -72,50 +69,23 @@ func OpenInMemory() (db *DB, err error) {
 	db = &DB{
 		DB: sqlite,
 	}
-	if err := db.init(); err != nil {
-		return nil, utils.CombineErrors(err, db.DB.Close())
-	}
 
 	return db, nil
 }
 
-func (db *DB) init() (err error) {
-	tx, err := db.DB.Begin()
-	if err != nil {
-		return err
-	}
-
-	defer func() { _ = tx.Rollback() }()
-
-	_, err = tx.Exec("CREATE TABLE IF NOT EXISTS `ttl` (`id` BLOB UNIQUE, `created` INT(10), `expires` INT(10), `size` INT(10));")
-	if err != nil {
-		return err
-	}
-
-	_, err = tx.Exec("CREATE TABLE IF NOT EXISTS `bandwidth_agreements` (`satellite` BLOB, `agreement` BLOB, `signature` BLOB);")
-	if err != nil {
-		return err
+// Migration define piecestore DB migration
+func Migration() *migrate.Migration {
+	migration := &migrate.Migration{
+		Table: "sn_versions",
+		OnCreate: migrate.SQL{
+			"CREATE TABLE IF NOT EXISTS `ttl` (`id` BLOB UNIQUE, `created` INT(10), `expires` INT(10), `size` INT(10));",
```

Comment:
These all should be `CREATE TABLE`, because we want exact behavior... it's easy to miss migration steps.